packages/fast-development-site-react/src/components/toc/toc-menu.tsx

```diff
@@ -24,8 +24,17 @@ const dropdownActive: string = "PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAx
 const dropdownInactive: string = "PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAzMiAzMiI+PHRpdGxlPmNoZXZyb24tcmlnaHQ8L3RpdGxlPjxwYXRoIGQ9Ik0yMi43OCwxNiw4LjI1LDEuNDUsOS42NSwwbDE2LDE2LTE2LDE2LTEuNC0xLjQxWiIvPjwvc3ZnPg==";
 
 const style: ComponentStyles<ITocMenuManagedClasses, IDevSiteDesignSystem> = {
-    toc__menu: {},
-    toc__menu__button: {
+    toc_menu: {},
+    toc_menu_button: {
+        fill: "none",
+        background: "none",
```

Comment:
do we need to specify background none? seems like it would be the default

Reply:
if this is a button I think to ensure that it works cross browser we'll need to add this. Native buttons tend to have a background of some sort applied.